### PR TITLE
Add Patreon

### PR DESCRIPTION
**Pull request by iandelahorne**
Adds the following events or companies:
 - Patreon

### Checklist

 - [X] I have updated the event or company counts
 - [X] I have put the most recent relevant date in the "Last Update" column
 - [] I have linked to the article about the change, not the company's website

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,7 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
 | Orbital Insight | Recommended | Restricted | Restricted | Restricted | 2020-03-11 |
 | Pandora | Optional | Restricted | Restricted | ? | 2020-03-09 |
 | Parler | Required | Restricted | Restricted | Restricted | 2020-03-06 |
+| Patreon | Required | Restricted | Resrictd | ? | 2020-03-11 |
```
Comment (drewbrew) [inline README.md]:
```suggestion
| Patreon | Required | Restricted | Restricted | ? | 2020-03-11 |
```

Typo

Comment (iandelahorne) [reply README.md]: Thanks, fixed!

**Comment by drewbrew** [comment]
Do you happen to have a public link? I'm guessing no but just want to make sure.

**Comment by iandelahorne** [comment]
> Do you happen to have a public link? I'm guessing no but just want to make sure.

@drewbrew I do not have a public link currently

**Pull request merged**